### DISCUSSION TITLE
Struggling to get region support working

### DISCUSSION THREAD

**Discussion by har-nick** [comahe-de/i18n4k#55] [Q&A]
Hi! I noticed one of the issues brought up region support, which was then fixed in 0.7.0. I'm using that version, but my strings don't appear unless I remove the region from my file name and locale list. Instead, they just appear as shown in the attached image.

<details>
<summary>My code:</summary>

#### build.gradle.kts

```kotlin
i18n4k {
    sourceCodeLocales = listOf("en_GB")
}
```

#### commonMain/i18n/onboarding/OnboardingStrings_en_GB.properties

```
welcomePageTitle=Welcome!
```
</details>

<details>
<summary>Resulting string</summary>

![Screenshot_20240108-184228_Trebuchet](https://github.com/comahe-de/i18n4k/assets/56137506/5e681878-ae91-4cf9-80db-6bf8e2974b1b)

</details>

**Comment by comahe-de** [comment]
Probably your active locale is "en", which will not default to "en_GB". It is the other way around: If you set the locale "en_GB", values of "en" will be used, if not found in "en_GB". Therefore, store your English messages in "en". If there are some special messages for "en_GB", set them additionally there.
